src/jormungandr/testing/jormungandr-automation/src/jormungandr/explorer/verifiers/vote_plan_verifier.rs

```diff
@@ -72,6 +72,9 @@ impl ExplorerVerifier {
             );
             match &vote_proposal_status.tally {
                 Tally::Public { result } => {
+                    if explorer_proposal.tally.is_none() {
+                        continue;
+                    }
```

Comment:
PublicTally cannot be None. Before the beginning of the tally time it will be all zeros `results: [0, 0, 0]` You can check the response of the voteplanstatus api So I would change this line of code with `assert!(explorer_proposal.tally.is_some())`

Reply:
the unwrap fails [here](https://github.com/input-output-hk/catalyst-core/blob/npg-3334-public-tally-fix/src/jormungandr/testing/jormungandr-automation/src/jormungandr/explorer/verifiers/vote_plan_verifier.rs#L79) without the continue. It seems the [explorer_vote_plan ](https://github.com/input-output-hk/catalyst-core/blob/npg-3334-public-tally-fix/src/jormungandr/testing/jormungandr-automation/src/jormungandr/explorer/verifiers/vote_plan_verifier.rs#L12) stores the public tally as None at origin. Is this incorrect behaviour?

Reply:
In my view yes it is incorrect. If you check the rest apis, in case the public tally is not started yet we return an array of 0 and the explorer should do the same to keep all our responses coherent with each other